cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/deployment/AbstractUndeployCommand.java

```diff
@@ -118,6 +118,7 @@ static void undeployName(CommandContext ctx, String name,
         try {
             ModelNode request = buildRequest(ctx,
                     name, allServerGroups, serverGroups, keepContent, headers);
+
```

Comment:
You can remove this new line.

Reply:
@bstansberry we are changing the behavior but in a manner that should be aligned with what a user expect.
Today, in standalone mode a check is done on the client to see if the deployment exists and is enabled. If the deployment doesn't exist or is already disabled, then an empty composite is sent and we got a generic "empty composite error" that is meaningless. This PR removes this client check and send the request in all cases.
To me that seems to be fine.

It would help to have a comment explaining what is the domain and new standalone logic there:
- In domain mode, when disabling a deployment from a server-group, the operation is sent to the server whatever the state of the deployment (no check is made). If the deployment is already disabled, there doesn't seem to have any side effect to disable an already disabled deployment. If the deployment doesn't exist, then the server will report a proper error.

- Similarly, in standalone mode, an operation to disable the deployment is sent whatever the fact that the deployment exists and/or is enabled. If the deployment is already disabled, there doesn't seem to have any side effect to disable an already disabled deployment. If the deployment doesn't exist, then the server will report a proper error.